sklearn/decomposition/tests/test_pca.py

```diff
@@ -72,7 +72,7 @@ def test_whitening(solver, copy):
     assert X.shape == (n_samples, n_features)
 
     # the component-wise variance is thus highly varying:
-    assert X.std(axis=0).std() > 43.8
+    assert X.std(axis=0).std() > 40
```

Comment:
The previous hardcoded threshold of `43.8` seemed to be tailored to `X` generated with seed 0. Reduced this a bit to pass for all seeds in the 0-99 range. Comparing with the new threshold of `40` still confirms highly varying variances.